qutip/solver/nm_mcsolve.py

```diff
@@ -323,13 +323,6 @@ class NonMarkovianMCSolver(MCSolver):
 
     options : SolverOptions, [optional]
         Options for the evolution.
-
-    seed : int, SeedSequence, list, [optional]
```

Comment:
Should the `seeds` option not be here?

Reply:
No seed is an input of the `run` method, not `__init__`.